week-10-angular/day-02/module-01/readme.md

```diff
@@ -1 +1,223 @@
 # <img src="https://cloud.githubusercontent.com/assets/7833470/10899314/63829980-8188-11e5-8cdd-4ded5bcb6e36.png" height="60"> Intro Angular
+
+| **Learning Objectives** |
+| :---- |
+| Grok Angular's philosophy |
+| Initialize Angular in an HTML view |
+| Experiment with templating & expressions |
+| Organize your code with controllers |
+| Pass data between the View & Controller with $scope|
+| Leverage built-in filters |
+| Leverage built-in directives |
+| Implement 2-way data binding |
+
+
+## Prereading Highlights
```

Comment:
Since we're not giving them pre-reading (at least I don't think we are), you might want to rename this section to "Angular Intro Highlights", or something like that.